subsys/logging/Kconfig.misc

```diff
@@ -65,4 +65,11 @@ config LOG_MEM_UTILIZATION
 	  When enabled, maximum usage of memory used for log messages in deferred
 	  mode is tracked. It can be used to trim LOG_BUFFER_SIZE.
 
+config LOG_CUSTOM_SECTION
+	bool "Allow custom log sections mapping"
+	help
+	  By default, logs are kept in ROM area. Enabling this configuration
+	  allows moving all the strings and backends to RAM area. This can be useful
+	  for architectures that have flash cache issues or not enough space in ROM.
+
```

Comment:
@nordic-krch @dcpleung PTAL this approach so that we can have custom log* sections placement.

Reply:
It works either way for me. Who else can suggest or approve this (or yours)?

Reply:
@nordic-krch may have some thoughts about this...

Reply:
Did not go on with this approach.